src/main/java/pl/edu/zut/mad/schedule/data/ScheduleRepository.kt

```diff
@@ -26,15 +26,14 @@ internal class ScheduleRepository(private val database: ScheduleDatabase, privat
             .subscribeOn(Schedulers.io())
             .subscribe()
 
-    fun getDayByDate(date: LocalDate): Observable<OptionalDay> =
-        Observable.fromCallable<OptionalDay> {
-            database.instance.where(DayApi::class.java)
-                .equalTo(DATE_COLUMN, date.toDate())
-                .findFirst()
-                ?.asFlowable<DayApi>()
-                ?.map { mapper.toDayUiFromApi(it) }
-                ?.blockingFirst() ?: EmptyDay(date)
-        }
+    fun getDayByDate(date: LocalDate): Observable<OptionalDay> {
```

Comment:
Is this tested?

Reply:
Check https://github.com/MADWI/schedule/pull/7/commits/66b8c58de98ec4d767bbedb68a0bc8d16f6bc1ce :)